tor/core/commands.json

```diff
@@ -0,0 +1,36 @@
+{
+  "notAuthorizedResponses": [
+    "Ha, nope!\n\n{}",
+    "l0l no.\n\n{}",
+    "{}",
+    "Sorry!\n\n{}",
+    "Nada.\n\n{}"
+  ],
+  "commands": {
+    "blacklist": {
+      "description": "Add usernames to internal blacklist - a.k.a do not interact with this user.",
+      "allowedNames": [],
+      "pythonFunction": "process_blacklist"
+    },
+    "reload": {
+      "description": "Re-run the initialization for the global config object and refresh all values.",
+      "allowedNames": [],
+      "pythonFunction": "reload_config"
+    },
+    "update": {
+      "description": "Update the local repo from master and respawn process.",
+      "allowedNames": [],
+      "pythonFunction": "update_and_restart"
+    },
+    "ping": {
+      "description": "Ping the bot to see if it's alive - user receives 'Pong!' response on success.",
+      "allowedNames": [
+        "jabbathehutt1234",
+        "carrillo232",
+        "FletcherAU",
+        "ILikePokemonGo101"
```

Comment:
We can do a check for which users we want to authorize for what commands, then do a PR for that later. Looks good for now.

Reply:
I just did 250+ transcriptions. (This is also just for !ping, so doesn't matter in the end)

Can you spot the odd one out?

Reply:
Not sure what that has to do with this, but 🤷‍♂️ . I was just trying to say not to worry about getting absolutely everyone on the whitelist immediately. We can handle that in a separate PR as we find more people to add to the list.